README.adoc

```diff
@@ -27,7 +27,7 @@ endif::[]
 image:https://img.shields.io/badge/maintained%20by-gruntwork.io-%235849a6.svg[link="https://gruntwork.io/?ref=repo_k8s_service"]
 
 This repo contains Helm Charts for deploying your applications on Kubernetes clusters with
-https://helm.sh[Helm].
+https://helm.sh[Helm] (hosted at [helmcharts.gruntwork.io](https://github.com/gruntwork-io/helmcharts)).
```

Comment:
NIT: It's a bit weird to list `helmcharts.gruntwork.io` as the URL but actually link to the GitHub repo. Perhaps make the first link actually go to `helmcharts.gruntwork.io`, but then add a second "implemented via https://github.com/gruntwork-io/helmcharts" link?

Reply:
Good call. Fixed in https://github.com/gruntwork-io/helm-kubernetes-services/pull/69. Embarrassingly, I also forgot this was asciidoc and not markdown so the link rendering is messed up right now 🤦